src/qt/sendcoinsdialog.h

```diff
@@ -100,13 +100,14 @@ private Q_SLOTS:
 };
 
 
+#define SEND_CONFIRM_DELAY   3
 
 class SendConfirmationDialog : public QMessageBox
 {
     Q_OBJECT
 
 public:
-    SendConfirmationDialog(const QString &title, const QString &text, int secDelay = 0, QWidget *parent = 0);
+    SendConfirmationDialog(const QString &title, const QString &text, int secDelay = SEND_CONFIRM_DELAY, QWidget *parent = 0);
```

Comment:
@ryanofsky: because setting `QWidget *parent = 0` is required, that would either result in swapping `QWidget *parent = 0` with `int secDelay = SEND_CONFIRM_DELAY` or passing `SEND_CONFIRM_DELAY` in `sendcoinsdialog.cpp` (as it is by now), right?
The current situation seems to be okay to me.
Or do I misunderstand something?